.travis.yml

```diff
@@ -8,21 +8,37 @@ before_install:
   - export MAJOR_VERSION="2"
   - export VSCODE_VERSION="1.39.2"
   - export VERSION="$MAJOR_VERSION.$TRAVIS_BUILD_NUMBER"
-  - export TAG="$VERSION-vsc$VSCODE_VERSION"
+  - export TAG="$VERSION-vsc$VSCODE_VERSION-$(uname -m)"
```

Comment:
I'm not sure we should append this here since that'd result in a separate tag for every version instead of having all the releases grouped under a single tag (e.g. https://github.com/cdr/code-server/releases/latest).

We could append it to the Docker tag only. There's also the v2 tag so we'd need to append to that as well. Should we leave `v2` pointing to amd64 and use `v2:arm64` for the arm64 version maybe? And then do the same for the version tag.

Reply:
Not sure if I follow, so you're telling me that `$TAG` is used as well as a version string?

Reply:
Yup, `$TAG` is used to create the git tag and is where the releases will get attached, like this one here: https://github.com/cdr/code-server/releases/tag/2.1523-vsc1.38.1

With `uname -m` appended we'd end up with separate tags for every tar/zip instead of having them under one tag.